server/alpine.Dockerfile

```diff
@@ -35,7 +35,7 @@ EXPOSE 9987/tcp 9987/udp 10101/tcp 30303/tcp
 VOLUME ["/ts/logs", "/ts/certs", "/ts/config", "/ts/files", "/ts/database", "/ts/crash_dumps"]
 
 ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/ts/libs/" \
-    SERVER_VERSION="${SERVER_VERSION}" \
+    SERVER_VERSION="${SERVER_VERSION:-latest-$(date +%d%m%y)}" \
```

Comment:
This will default to `latest-<buildDate>` if SERVER_VERSION is not set (aka if you build the latest and not specify a version via the `--build-arg` option

IMO that should be fine as this ENV won't get used by anyone who builds the image themselves whilst not setting the version